server/src/test/java/org/candlepin/model/ConsumerTest.java

```diff
@@ -414,32 +414,32 @@ public void testLookupUsersConsumer() {
         assertEquals(consumer, consumerCurator.findByUser(user));
     }
 
-    @Test
-    public void testConsumerFactsFilter() {
-        String oldValue = config.getString(ConfigProperties.CONSUMER_FACTS_MATCHER);
-        config.setProperty(ConfigProperties.CONSUMER_FACTS_MATCHER, "^goodkey.*");
+    // @Test
+    // public void testConsumerFactsFilter() {
+    //     String oldValue = config.getString(ConfigProperties.CONSUMER_FACTS_MATCHER);
+    //     config.setProperty(ConfigProperties.CONSUMER_FACTS_MATCHER, "^goodkey.*");
 
-        Consumer consumer = new Consumer("a consumer", "username", owner, consumerType);
+    //     Consumer consumer = new Consumer("a consumer", "username", owner, consumerType);
 
-        Map<String, String> facts = new HashMap<String, String>();
-        facts.put("badkey.something", "zaz");
-        facts.put("goodkey.something", "foobar");
+    //     Map<String, String> facts = new HashMap<String, String>();
+    //     facts.put("badkey.something", "zaz");
+    //     facts.put("goodkey.something", "foobar");
 
-        consumer.setFacts(facts);
+    //     consumer.setFacts(facts);
 
-        consumer = consumerCurator.create(consumer);
+    //     consumer = consumerCurator.create(consumer);
 
-        assertNull(consumer.getFact("badkey.something"));
-        assertEquals("foobar", consumer.getFact("goodkey.something"));
+    //     assertNull(consumer.getFact("badkey.something"));
+    //     assertEquals("foobar", consumer.getFact("goodkey.something"));
 
-        consumer.setFact("anotherbadkey", "zippy");
+    //     consumer.setFact("anotherbadkey", "zippy");
 
-        consumer = consumerCurator.update(consumer);
+    //     consumer = consumerCurator.update(consumer);
 
-        assertNull(consumer.getFact("anotherbadkey"));
+    //     assertNull(consumer.getFact("anotherbadkey"));
 
-        config.setProperty(ConfigProperties.CONSUMER_FACTS_MATCHER, oldValue);
-    }
+    //     config.setProperty(ConfigProperties.CONSUMER_FACTS_MATCHER, oldValue);
+    // }
```

Comment:
Commented out test. I think this was in 49 as well, but since it has been merged, lets not worry about it in the 49 branch.